docs/dev/how-to/verify-contracts.md

```diff
@@ -1,3 +1,10 @@
+---
+head:
+  - - meta
+    - name: "twitter:title"
+      content: Verify Contracts with Hardhat | zkSync Era docs
```

Comment:
I think we should be consistent in how we capitalise titles, in English subtitles and titles have capital letters for almost every word like with this title. I will suggest changes for the previous titles